spec/emittr_spec.rb

```diff
@@ -77,6 +77,29 @@
     end
   end
 
+  describe '#once' do
+    let(:callback) { Proc.new {} }
+
+    before { emitter.once :once_test, &callback }
+
+    it 'calls callback' do
```

Comment:
Maybe we could add a `context` here? Like `context 'when event is triggered'`.

We should also test if the callback is called only once. What about  putting an `once` on the `receive` and emit twice to make sure it's emitted only once?